presto-tests/src/main/java/com/facebook/presto/tests/sql/JdbcSqlExecutor.java

```diff
@@ -17,21 +17,32 @@
 import java.sql.DriverManager;
 import java.sql.SQLException;
 import java.sql.Statement;
+import java.util.Properties;
+
+import static java.util.Objects.requireNonNull;
 
 public class JdbcSqlExecutor
         implements SqlExecutor
 {
     private final String jdbcUrl;
+    private final Properties jdbcProperties;
 
     public JdbcSqlExecutor(String jdbcUrl)
     {
-        this.jdbcUrl = jdbcUrl;
+        this(jdbcUrl, new Properties());
+    }
+
+    public JdbcSqlExecutor(String jdbcUrl, Properties jdbcProperties)
```

Comment:
Some might want to remove this change as this constructor is not used in the Presto codebase... I wonder if there is a way to somehow mark that it is used alsewhere

Reply:
We could introduce some annotation like `@ExternalProjectDependency` or sth like that. It would be helpful in other places too, like `JdbcClient.getConnection(JdbcSplit)`. @electrum what do you think?